go.mod

```diff
@@ -1,19 +1,18 @@
 module github.com/celestiaorg/knuu
 
-go 1.22.4
+go 1.22.5
 
 require (
 	github.com/celestiaorg/bittwister v0.0.0-20231213180407-65cdbaf5b8c7
 	github.com/docker/docker v26.1.4+incompatible
 	github.com/google/uuid v1.6.0
-	github.com/joho/godotenv v1.5.1
 	github.com/minio/minio-go/v7 v7.0.73
 	github.com/sirupsen/logrus v1.9.3
 	github.com/stretchr/testify v1.9.0
 	gopkg.in/yaml.v3 v3.0.1
-	k8s.io/api v0.28.2
-	k8s.io/apimachinery v0.28.2
-	k8s.io/client-go v0.28.2
+	k8s.io/api v0.30.2
+	k8s.io/apimachinery v0.30.2
+	k8s.io/client-go v0.30.2
```

Comment:
This will break `chaos-mesh`.
But as it is not in use at the moment, we should be fine doing that.